### ISSUE TITLE
I can't find rule-path

### ISSUE THREAD

**Issue by jelly-pawpaw** [derenlei/KG-RuleGuider#2]
What is rule-path?
![QQ截图20201123195135](https://user-images.githubusercontent.com/71800844/99959104-6585b200-2dc5-11eb-839a-c11012fa0805.png)

**Comment by Destiny-Suk** [comment]
Hi, I also have this question. Have you solved it now?

**Comment by jelly-pawpaw** [comment]
No,I have not resolved.I try to run the code in paper AnyBURL,but the format of output doesn't meet requirement.




















At 2020-11-27 20:24:06, "Destiny-Suk" <notifications@github.com> wrote:

Hi, I also have this problem. Have you solved it now?

**Comment by qiaoxiaoqiao1** [comment]
Anybody solved this issue?

**Comment by qiaoxiaoqiao1** [comment]
I figured it out "data/FB15K-237/adj_list.pkl"

**Comment by BlackLucy** [comment]
Hi, Do you know which file  "pretrain-checkpoint-path "  is ?

**Comment by qiaoxiaoqiao1** [comment]
> Hi, Do you know which file "pretrain-checkpoint-path " is ?
checkpoint_path FB15K-237—conve-RV-xavier-200-200-0.003-32-3-0.3-0.2-0.1/model_best.tar

**Comment by BlackLucy** [comment]
> > Hi, Do you know which file "pretrain-checkpoint-path " is ?
> > checkpoint_path FB15K-237—conve-RV-xavier-200-200-0.003-32-3-0.3-0.2-0.1/model_best.tar


Hello, is this file ‘checkpoint_path FB15K-237—conve-RV-xavier-200-200-0.003-32-3-0.3-0.2-0.1/model_best.tar’in the KG-RuleGuider project? I did not find the file you mentioned. 
![image](https://user-images.githubusercontent.com/30290790/119069153-29e83f80-ba18-11eb-8d6b-0a9b758c1cfc.png)

**Comment by BlackLucy** [comment]
![image](https://user-images.githubusercontent.com/30290790/119428497-92e4f580-bd3f-11eb-82f5-e79716208537.png)
Excuse me, do you know how to obtain these files?

**Comment by summerone123** [comment]
Excuse me, do you know how to get the rule and use it, I have run the code in paper AnyBURL,but the format of output doesn't meet requirement.

**Comment by summerone123** [comment]
> 
> No,I have not resolved.I try to run the code in paper AnyBURL,but the format of output doesn't meet requirement. At 2020-11-27 20:24:06, "Destiny-Suk" <notifications@github.com> wrote: Hi, I also have this problem. Have you solved it now? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

hello，have you solve the issue?
